burrow/Cargo.toml

```diff
@@ -7,5 +7,7 @@ edition = "2021"
 crate-type = ["lib", "staticlib"]
 
 [dependencies]
+ipnetwork = "0.20.0"
+rtnetlink = "0.12.0"
```

Comment:
Curious why `burrow` needs rtnetlink and ipnetwork?

Reply:
Forgot to remove those; thanks!

Reply:
https://docs.rs/rtnetlink/latest/rtnetlink/struct.LinkSetRequest.html#method.up

Look at using this method instead